### ISSUE TITLE
rsyslog lfmerge errors in app container

### ISSUE THREAD

**Issue by megahirt** [sillsdev/web-languageforge#1055]
According to [comments in 1034](https://github.com/sillsdev/web-languageforge/pull/1034#discussion_r683710223) we are seeing:

- errors in the init container related to not having which installed
- errors in the app container related to rsyslog.  I don't know exactly what it means:
```
rsyslogd: activation of module imklog failed [v8.1901.0 try https://www.rsyslog.com/e/2145 ]
```
Looks like https://stackoverflow.com/a/60265997 has an acceptable way to comment out the imklog module from rsyslog.

**Comment by rmunn** [comment]
The `which` issue is fixed by #1059. Turns out that `which` is installed, I just didn't use it properly in the `if` statement. (The brackets weren't needed.)

I'll submit another PR for the imklog issue.

**Comment by rmunn** [comment]
Changed my mind and put both fixes into #1059, so both of these issues can be fixed with a single PR.
